### ISSUE TITLE
pkg.installed doesn't work on Manjaro.

### ISSUE THREAD

**Issue by crocket** [saltstack/salt#31788]
### Description of Issue/Question

I installed salt-git from AUR. It fetched the latest git commit and built SaltStack and turned it into a package. When I execute `salt-call --local state.highstate`, I see the error message below for many sls files.

```
----------
          ID: zsh
    Function: pkg.installed
      Result: False
     Comment: State 'pkg.installed' was not found in SLS 'pkg.zsh'
              Reason: 'pkg' __virtual__ returned False
     Started: 
    Duration: 
     Changes:
```

All error messages are related to pkg.installed

### Steps to Reproduce Issue

Test pkg.installed on Manjaro.

### Versions Report

`salt-call 2016.3.0-190-g538ec98 (Boron)`

**Comment by crocket** [comment]
```python
>> salt.grains.core._parse_os_release()
{'NAME': 'Manjaro Linux', 'ANSI_COLOR': '1;32', 'SUPPORT_URL': 'http://www.manjaro.org/', 'BUG_REPORT_URL': 'http://bugs.manjaro.org/', 'PRETTY_NAME': 'Manjaro Linux', 'HOME_URL': 'http://www.manjaro.org/', 'ID': 'manjaro'}
```

**Comment by gtmanfred** [comment]
this should already be done.

https://github.com/saltstack/salt/pull/27841/files

It should appear as Manjaro as the os grain, so you just need to add it to this list.

https://github.com/saltstack/salt/blob/develop/salt/modules/pacman.py#L32

**Comment by crocket** [comment]
`grain['os']` was `ManjaroLinux`
`grain['os_family']` was initially `ManjaroLinux`, but it somehow became `Arch` after doing various things with SaltStack.

However, `grain['os_family']` doesn't affect how `pacman.py` works.

**Comment by rallytime** [comment]
@crocket Does the fix in #32002 resolve this issue for you?

**Comment by crocket** [comment]
@rallytime Unfortunately, no. I don't know how grains['os'] is calculated. It seems unstable. It was `ManjaroLinux` before, but it is now `Manjaro`.

```
$ salt-call --local grains.items
os:
    Manjaro
os_family:
    Arch
osarch:
    x86_64
oscodename:
    Capella
osfullname:
    ManjaroLinux
osrelease:
    15.12
```

**Comment by crocket** [comment]
To properly test it, I removed `/etc/salt/grains` and installed salt-git from AUR. The version installed by salt-git is `2016.3.0-272-g65abe44 (Boron)`.

I tested `pkg.installed` right after installing salt-git, and it didn't work.
I manually added `ManjaroLinux` to the tuple in `__virtual__` in `modules/pacman.py`, and `pkg.installed` started working.

Why is grains['os'] `Manjaro` on one version and `ManjaroLinux` on another version?
